integration-tests/plugins/__tests__/inventory/inventory-items/index.js

```diff
@@ -538,6 +538,44 @@ describe("Inventory Items endpoints", () => {
           })
         )
       })
+
+      it("Lists inventory items searching by title, description and sku", async () => {
+        const api = useApi()
+
+        const inventoryService = appContainer.resolve("inventoryService")
+
+        await Promise.all([
+          inventoryService.createInventoryItem({
+            title: "Test Item",
+          }),
+          inventoryService.createInventoryItem({
+            description: "Test Desc",
+          }),
+          inventoryService.createInventoryItem({
+            sku: "Test Sku",
+          }),
+        ])
+
+        const response = await api.get(
+          `/admin/inventory-items?q=test`,
+          adminHeaders
+        )
+
+        expect(response.data.inventory_items).toHaveLength(3)
+        expect(response.data.inventory_items).toEqual(
+          expect.arrayContaining([
```

Comment:
todo: Please add tests ensuring that items we don't expect are not included in the output

Reply:
👍 tested to ensure that the default is not part of the result. 

the expect on 564 should ensure that only the three objects are part of the result, the arrayContaining is just to ensure ordering doesn't matter